README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.com/bids-standard/bids-specification.svg?branch=master)](https://travis-ci.com/bids-standard/bids-specification)
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-specification.svg?style=svg)](https://circleci.com/gh/bids-standard/bids-specification)
+![](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)
```

Comment:
```suggestion
[![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard]
```

Reply:
I applied this change on my test directory and the rendering may not be what was intended - https://github.com/franklin-feingold/bids-specification-test/blob/master/README.md

Here is the previous rendering on my fork - https://github.com/franklin-feingold/bids-specification/blob/readme/README.md

Reply:
I see. I don't know how to make this shield work, then, but it doesn't seem very useful to create one that doesn't link to the Twitter account.

Reply:
Oh, there was a typo in my suggestion. Fixed.

Reply:
looks like it works now, thank you!